src/test/acceptance/features/do-you-live-in-scotland.feature

```diff
@@ -24,3 +24,10 @@ Feature: Do you live in Scotland?
     And I click continue
     Then I am shown the I live in Scotland page
     And I am informed that I cannot apply and should use the Scottish scheme
+
+  Scenario: I can change my answer to yes from the check details page
```

Comment:
You should probably mention which question you're changing to yes in the Scenario title.